<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager_widget</class>
 <widget class="QWidget" name="manager_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>经理界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="back_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/image/images/manager.png</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>441</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>服务员</string>
    </attribute>
    <widget class="QTableWidget" name="waiter_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>417</width>
       <height>411</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>个人编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>服务桌号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>相应评价</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>厨师</string>
    </attribute>
    <widget class="QTableWidget" name="chef_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>417</width>
       <height>411</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>个人编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>菜品编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>相应评价</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="query_pushButton">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>64</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="black_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(57, 57, 57);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(44, 90, 255, 230);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>821</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>13</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(71, 143, 214);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <zorder>label_14</zorder>
  <zorder>tabWidget</zorder>
  <zorder>black_label</zorder>
  <zorder>label_13</zorder>
  <zorder>query_pushButton</zorder>
  <zorder>back_pushButton</zorder>
  <zorder>label_3</zorder>
  <zorder>lcdNumber</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
